sanic/sanic.py

```diff
@@ -313,6 +299,60 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
 
         log.info("Server Stopped")
 
+    def run_with_reload(self, **dataset):
+        # Run reloader thread
+        event_loop = start_reloader_thread(self, interval=1)
+
```

Comment:
Can we remove the spaces in between the statements here? More of a style thing than anything else but I feel as though they're not needed.